dingomata/cogs/text/commands.py

```diff
@@ -201,15 +201,21 @@ async def smooch(self, ctx: SlashContext, user: User) -> None:
            )
     @cooldown(1, 10.0, BucketType.member)
     async def cuddle(self, ctx: SlashContext, user: User) -> None:
-        await ctx.send(f'{ctx.author.display_name} pulls {self._mention(ctx, user)} into their arm for a long cuddle.')
+        if ctx.author == user:
+            await ctx.send(f"{ctx.author.display_name} can't find anyone to cuddle, so they decided to pull their tail in front and cuddles it instead.")
+        else:
+            await ctx.send(f'{ctx.author.display_name} pulls {self._mention(ctx, user)} into their arm for a long cuddle.')
 
     @slash(name='snug', guild_ids=service_config.get_command_guilds('snug'),
            description="Give someone some snuggles",
            options=[create_option(name='user', description='Target user', option_type=User, required=True)],
            )
     @cooldown(1, 10.0, BucketType.member)
     async def snug(self, ctx: SlashContext, user: User) -> None:
-        await ctx.send(f'{ctx.author.display_name} snuggles the heck out of {self._mention(ctx, user)}!')
+        if ctx.author == user:
+            await ctx.send(f"{ctx.author.display_name} can't find a hot wherewolf boyfriend to snuggle, so they decide to snuggles a daki with themselves on it.")
```

Comment:
wait just realized also in this one `to snuggles`

Reply:
snuggles? I thought that was spelled right wdingGoob